### PR TITLE
Add information about the physical coordinate space.

### PR DESCRIPTION
**Pull request by ursg**
This will be required both for physical initialization of simulations
directly inside the FSgrid, as well as direct simulation output, and
visualization grid specification.

### CODE REVIEW

**Comment by ykempf** [comment]
So should we add a set function and make that physicalGlobalStart a private member?

File: fsgrid.hpp [hunk 1]
```diff
@@ -765,10 +765,27 @@ template <typename T, int stencil> class FsGrid {
...
+       * the given cell.
+       *
+       * \param x local x-Coordinate, in cells
+       * \param y local y-Coordinate, in cells
+       * \param z local z-Coordinate, in cells
+       */
+      std::array<double, 3> getPhysicalCoords(int x, int y, int z) {
+         std::array<double, 3> coords;
+         coords[0] = physicalGlobalStart[0] + (localStart[0]+x)*DX;
+         coords[1] = physicalGlobalStart[1] + (localStart[1]+x)*DY;
```
Comment (ykempf) [inline fsgrid.hpp]: Spot the bug!

File: fsgrid.hpp [hunk 2]
```diff
@@ -765,10 +765,27 @@ template <typename T, int stencil> class FsGrid {
...
+         coords[2] = physicalGlobalStart[2] + (localStart[2]+x)*DZ;
```
Comment (ykempf) [inline fsgrid.hpp]: Spot the bug!

File: fsgrid.hpp (hunk 1 above)
Comment (ursg) [reply fsgrid.hpp]: Ah, x y and z... whoopsie!

**Comment by ykempf** [comment]
I'm good with this I think, but it's not fire-tested yet.

**Comment by ykempf** [comment]
Merging this as it worked in my tests.

**Pull request merged**